ASF/doap_Superset.rdf

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl"?>
+<rdf:RDF xml:lang="en"
+         xmlns="http://usefulinc.com/ns/doap#"
+         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:asfext="http://projects.apache.org/ns/asfext#"
+         xmlns:foaf="http://xmlns.com/foaf/0.1/">
+<!--
```

Comment:
I think this should be moved to the top.

Reply:
Bumping apache-rat (the license checking thing) from 0.13 to 0.16.1 seems to have cleared it. Looks from their changelog like it must've fixed a bunch of other things too, like testing for those google-hosted fonts we're not supposed to use.